content/authorization/application-registration-prerequisites.md

```diff
@@ -0,0 +1,207 @@
+# Application Registration Prerequisites / Developer Guide
```

Comment:
Is this prerequisite for patient apps / direct to consumers? If so, shall we mention that in the title?

Reply:
@kolkheang - While it's true that the Trust Framework is currently only applied to Patient facing applications.  All the best practices listed here would be benign at worst and often beneficial for Provider facing applications, too.  I'd be inclined to leave the advice as general purpose, but perhaps mention in the FAQ that it's not (yet) being applied to provider-facing apps.